vite.config.ts

```diff
@@ -5,22 +5,17 @@ import react from "@vitejs/plugin-react";
 export default defineConfig({
   server: {
     proxy: {
-      "/ws": {
-        target: "https://ngx.ampath.or.ke/amrs",
-        changeOrigin: true,
-        secure: false,
-      },
-      "/api": {
-        target: "http://0.0.0.0:5080",
+      "/openmrs": {
+        target: "http://localhost:8089/",
```

Comment:
For this project `openmrs` is part of the base URL
```
{
"/ws": {
        target: "http://localhost:8089/openmrs",
        changeOrigin: true,
        secure: false,
      },
}
```

Reply:
:+1: